src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -1282,6 +1282,116 @@ and ``live_collection_button_delete`` respectively:
         {{ block('button_widget') }}
     {% endblock live_collection_button_add_widget %}
 
+If you only want to customize some attributes maybe simpler to use the options in the form type:
+
+    // ...
+    ->add('comments', LiveCollectionType::class, [
+        'entry_type' => CommentFormType::class,
+        'allow_add' => true,
+        'allow_delete' => true,
+        'by_reference' => false,
+        'label' => false,
+        'button_delete_options' => [
+            'label' => 'X',
+            'attr' => [
+                'class' => 'btn btn-outline-danger',
+            ],
+        ]
+    ])
+    // ...
+
+If you want more control over how each row is rendered you can override the blocks
+related to the ``LiveCollectionType``. This works the same way as for `the traditional
+collection type`_, but you should use ``live_collection`` and ``live_collection_entry`` as
+prefixes instead.
+
+Let's continue our previous example and customize the rendering of the blog post comments form.
+By default the add comment button is placed after the comments, let's move it before them.
+
+.. code-block:: twig
+
+    {%- block _blog_post_form_comments_widget -%}
```

Comment:
Should this `live_collection_widget`? I mean, I know what you have is correct for this situation, but I think showing `live_collection_widget` would be better (it would be very rare someone would need to override `LiveCollectionType` twice in the same form in a different way).

Reply:
I meant this could be an example for customizing just for a specific form type, and the next with the bootstrap table could be a generic one. But it's fine by me to keep it simple and just use `live_collection_widget` here too.